ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/helpcenter/searcharticlepage/SearchArticlePresenterTest.java

```diff
@@ -0,0 +1,297 @@
+package com.kayako.sdk.android.k5.helpcenter.searcharticlepage;
+
+import com.kayako.sdk.android.k5.common.adapter.BaseListItem;
+import com.kayako.sdk.android.k5.common.adapter.searchlist.SearchListItem;
+import com.kayako.sdk.helpcenter.search.SearchArticle;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+public class SearchArticlePresenterTest {
+    private SearchArticlePresenter searchArticlePresenter;
+
+    @Mock
+    private SearchArticleContract.View mView;
+
+    @Mock
+    private SearchArticleContract.Data mData;
+
+    @Mock
+    private SearchArticle searchArticle;
+
+    private final static String TEST_VALUE = "test";
```

Comment:
⚠️  The order of modifier specification should follow the Java standard style guide as in the proposed solution. Likewise, the field declarations should be done in the order of visibility, then by modifier. This suggestion will add the advantage that other people will be able to read your code more easily.  

**[More details](https://confluence.devfactory.com/x/VY8-Ew)**
          
## The possible solution in this case

```JAVA
private static final String TEST_VALUE = "test";

```
 

<a id="50f1dda0-52d5-11e8-9c23-69d6096f3efc" href="none" codereview-uuid-reserved-tag />